pkg/executor/common.go

```diff
@@ -327,22 +402,38 @@ func NewTemplatesFromEnv(env string) (t Templates, err error) {
 	return t, nil
 }
 
-// SyncDefaultExecutors creates or updates default executors
-func SyncDefaultExecutors(executorsClient executorsclientv1.Interface, namespace, data string, readOnlyExecutors bool) (
-	images Images, err error) {
-	var executors []testkube.ExecutorDetails
+func ParseExecutors(cfg *config.Config) (executors []testkube.ExecutorDetails, err error) {
+	var data []byte
 
-	if data == "" {
-		return images, nil
+	f, err := os.Open(filepath.Join(cfg.TestkubeConfigDir, "executors.json"))
```

Comment:
executors.json is dynamically updated in helm-chart for new executor relase.how are going to sync them?

Reply:
the `config` folder in the testkube serves as example config and for localdev.

idea is to use a ConfigMap and define executors.json, templates, etc etc... in the ConfigMap and mount it in the Testkube API Server pod.

when there is a change we will update it in the Helm chart, we could probably just use the Helm file function to add it in the ConfigMap, so we keep the already existing files

Reply:
ok, then makes sense